src/Lykke.Job.TxDetector/Modules/SagaModule.cs

```diff
@@ -0,0 +1,139 @@
+
+                    Register.BoundedContext("transactions")
+                        .FailedCommandRetryDelay(defaultRetryDelay)
+                        .ListeningCommands(typeof(ProcessTransactionCommand))
+                            .On("transactions-commands")
+                        .PublishingEvents(typeof(TransferOperationCreatedEvent), typeof(CashInOperationCreatedEvent))
+                            .With("transactions-events")
+                        .WithCommandsHandler<TransactionHandler>(),
+
+                    Register.BoundedContext("transfer")
+                        .FailedCommandRetryDelay(defaultRetryDelay)
+                        .ListeningCommands(typeof(ProcessTransferCommand))
+                            .On("transfer-commands")
+                        .PublishingEvents(typeof(TransferProcessedEvent))
+                            .With("transfer-events")
+                        .WithCommandsHandler<TransferHandler>(),
+
+                    Register.BoundedContext("cachein")
+                        .FailedCommandRetryDelay(defaultRetryDelay)
+                        .ListeningCommands(typeof(RegisterCachInOutCommand), typeof(ProcessCashInCommand))
+                            .On("cachein-commands")
+                        .PublishingEvents(typeof(CashInOutOperationRegisteredEvent), typeof(TransactionProcessedEvent))
+                            .With("cachein-events")
+                        .WithCommandsHandler<CashInHandler>(),
+
+                    Register.BoundedContext("notifications")
+                        .FailedCommandRetryDelay(defaultRetryDelay)
+                        .ListeningCommands(typeof(SendNotificationCommand))
+                            .On("notifications-commands")
+                        .WithCommandsHandler<NotificationsHandler>(),
+
+                    Register.BoundedContext("email")
+                        .FailedCommandRetryDelay(defaultRetryDelay)
+                        .ListeningCommands(typeof(SendNoRefundDepositDoneMailCommand))
+                            .On("email-commands")
+                        .WithCommandsHandler<EmailHandler>(),
+
+                    Register.BoundedContext("history")
+                        .ListeningEvents(typeof(TransferProcessedEvent))
+                            .From("transfer").On("transfer-events")
+                        .WithProjection(projection, "transfer"),
+
+                    Register.Saga<ConfirmationsSaga>("transactions-saga")
+                        .ListeningEvents(typeof(TransferOperationCreatedEvent), typeof(CashInOperationCreatedEvent))
+                            .From("transactions").On("transactions-events")
+                        .ListeningEvents(typeof(CashInOutOperationRegisteredEvent), typeof(TransactionProcessedEvent))
+                            .From("cachein").On("cachein-events")
```

Comment:
please seach for cachein :) and replace with cash in. I might miss smth

Reply:
Sorry, it is my main typo problem. Cash, cache, cashe.